GPU/GPUTracking/TRDTracking/GPUTRDTracker.cxx

```diff
@@ -200,6 +200,54 @@ void GPUTRDTracker_t<TRDTRK, PROP>::Reset()
   mNTracks = 0;
 }
 
+template <class TRDTRK, class PROP>
```

Comment:
Thx, I would propose to call the preparefunction from the DoTracking to avoid duplicating the code.